changelog.md

```diff
@@ -12,6 +12,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * The file is now deleted when the last component of this file is deleted.
 * Fix console error on reloading Model view page.
 
+### Changed
+
+* UI improvements:
+  * Component list:
+    * Don't show plugin name in accordion header
```

Comment:
Replace plugin name by "Components" in expansion item title

Reply:
done